icu_benchmarks/models/wrappers.py

```
@@ -104,6 +106,70 @@ def check_supported_runmode(self, runmode: RunMode):
+        else:
+            pred_indicators = np.hstack((pred_indicators.reshape(-1, 1), test_label.reshape(-1, 1)))
+            logging.info(pred_indicators.shape)
+            pred_indicators = np.hstack((pred_indicators, test_pred))
+            np.savetxt(
+                Path(self.logger.save_dir) / "pred_indicators.csv",
+                pred_indicators,
+                delimiter=",",
+                header="id,ground_truth,prediction_0,prediction_1",
+                fmt="%d,%d,%0.3f,%0.3f",
+            )
+            logging.debug(f"Saved row indicators to {Path(self.logger.save_dir) / f'row_indicators.csv'}")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)